src/printer.js

```diff
@@ -2206,20 +2221,32 @@ function printJSXElement(path, options, print) {
   let forcedBreak = false;
 
   // Trim trailing whitespace, recording if there was a hardline
+  let numTrailing = 0;
```

Comment:
The printer will automatically trim trailing whitespace. Are you sure you need to do this?